File: 백준/Silver/13241. 최소공배수/최소공배수.swift
// [func(Int, Int) -> Int] getGCD: 두 자연수를 입력 받아 최대 공약수를 출력하는 함수
func getGCD(_ a: Int, _ b: Int) -> Int {
    if (b == 0) {
        return a
    } else {
        return getGCD(b, a % b)
    }
}

// [func(Int, Int) -> Int] getLCM: 두 자연수를 입력 받아 최소 공배수를 출력하는 함수
func getLCM(_ a: Int, _ b: Int) -> Int {
    return (a*b) / getGCD(a, b)
}

// [Int] A, B (immutable): 최소 공배수를 계산할 두 자연수(범위(자연수): < 100,000,000)
let arrayOfInput: Array<Int> = (readLine() ?? "1 1").split(separator: " ").compactMap { Int($0) }
let (A, B): (Int, Int) = (arrayOfInput[0], arrayOfInput[1])

// getLCM 함수의 반환 값을 출력
print(getLCM(A, B))

File: 백준/Silver/13241. 최소공배수/최소공배수.py
# [def(int, int) -> int] getGCD: 두 자연수를 입력 받아서 최대 공약수를 반환하는 함수
def getGCD(a, b):
    if (b==0):
        return a
    else:
        return getGCD(b, a%b)

# [def(int, int) -> int] getLCM: 두 자연수를 입력 받아서 최소 공배수를 반환하는 함수
def getLCM(a, b):
    return (a*b) // getGCD(a, b)

# [int] A, B: 최소 공배수를 계산할 두 정수 (범위(정수): < 100,000,000)
A, B = map(int, input().split())

# getLCM 함수를 통해서 답 출력
print(getLCM(A, B))